WORKSPACE

```diff
@@ -74,42 +68,33 @@ http_archive(
     ],
 )
 
-http_archive(
-    name = "bazel_skylib",
-    sha256 = "1dde365491125a3db70731e25658dfdd3bc5dbdfd11b840b3e987ecf043c7ca0",
-    urls = [
-        "https://storage.googleapis.com/mirror.tensorflow.org/github.com/bazelbuild/bazel-skylib/releases/download/0.9.0/bazel_skylib-0.9.0.tar.gz",
-        "https://github.com/bazelbuild/bazel-skylib/releases/download/0.9.0/bazel_skylib-0.9.0.tar.gz",
-    ],
-)  # https://github.com/bazelbuild/bazel-skylib/releases
-# END: Upstream TensorFlow dependencies
+# Load tf_repositories() before loading dependencies for other repository so
+# that dependencies like com_google_protobuf won't be overridden.
+load("@org_tensorflow//tensorflow:workspace.bzl", "tf_repositories")
 
-# Apple and Swift rules.
-http_archive(
-    name = "build_bazel_rules_apple",
-    sha256 = "a045a436b642c70fb0c10ca84ff0fd2dcbd59cc89100d597a61e8374afafb366",
-    urls = ["https://github.com/bazelbuild/rules_apple/releases/download/0.18.0/rules_apple.0.18.0.tar.gz"],
-)  # https://github.com/bazelbuild/rules_apple/releases
+# Please add all new TensorFlow dependencies in workspace.bzl.
+tf_repositories()
 
-http_archive(
-    name = "build_bazel_rules_swift",
-    sha256 = "18cd4df4e410b0439a4935f9ca035bd979993d42372ba79e7f2d4fafe9596ef0",
-    urls = ["https://github.com/bazelbuild/rules_swift/releases/download/0.12.1/rules_swift.0.12.1.tar.gz"],
-)  # https://github.com/bazelbuild/rules_swift/releases
+register_toolchains("@local_config_python//:py_toolchain")
```

Comment:
Are we using this toolchain anywhere?

Reply:
I don't believe so, I copied it from the TF `WORKSPACE` but will remove it.